src/__tests__/Carousel.tsx

```diff
@@ -43,8 +43,8 @@ describe('Slider', function() {
 
         componentInstance = component.instance();
 
-        totalChildren = children && children.length ? React.Children.count(componentInstance.props.children) : 0;
-        lastItemIndex = totalChildren - 1;
+        totalChildren = children ? React.Children.count(componentInstance.props.children) : 0;
+        lastItemIndex = Math.max(totalChildren - 1, 0);
```

Comment:
Assuming that the correct logic is to not expect negative indices (e.g. in case of 0)

Reply:
Both of these changes are minor and could be skipped